lib/checks/text-alternatives/link-text.json

```diff
@@ -0,0 +1,11 @@
+{
+  "id": "link-text",
+  "evaluate": "link-text.js",
+  "metadata": {
+    "impact": "moderate",
+    "messages": {
+      "pass": "Link have an uniqe text and title",
```

Comment:
This could use some spelling and grammar help:

`Link has unique text and title content`